server/db/migration/V20220413.2005__adjust_tooltips_1059.sql

```diff
@@ -0,0 +1,112 @@
+update CalculationRule set 
+choices = '[{"id": "", "name": "N/A"},
+{"id": "1", "name": "20% of State Density Bonus"},
+{"id": "2", "name": "TOC Tiers 1-3"},
+{"id": "3", "name": "TOC Tier 4"},
+{"id": "4", "name": "100% Affordable"}]',
```

Comment:
These lines are not necessary to update the description. They can be safely deleted.
```suggestion
```